src/Classes/ModStore.lua

```diff
@@ -501,8 +501,36 @@ function ModStoreClass:EvalMod(mod, cfg)
 				return
 			end
 		elseif tag.type == "SocketedIn" then
-			if not cfg or tag.slotName ~= cfg.slotName or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
+			if not cfg or tag.slotName ~= cfg.slotName or (tag.socketColor ~= nil and (tag.sockets == nil or (tag.sockets ~= nil and type(tag.sockets) ~= "string")) and tag.socketColor ~= cfg.socketColor) or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
 				return
+			elseif tag.sockets ~= nil and tag.sockets == "all" and tag.socketColor ~= nil then
+					local targetAtrColor = tag.socketColor == "R" and "strengthGems" or tag.socketColor == "G" and "dexterityGems" or tag.socketColor == "B" and "intelligenceGems"
+					local total = (cfg["intelligenceGems"] or 0) + (cfg["dexterityGems"] or 0) + (cfg["strengthGems"] or 0)
+					local count = cfg[targetAtrColor] or 0
+					if total == 0 or count == 0 or total > count then
+						return
+					end
+			elseif tag.sockets ~= nil and type(tag.sockets) == "number" and tag.socketColor ~= nil then
+					local targetAtrColor = tag.socketColor == "R" and "strengthGems" or tag.socketColor == "G" and "dexterityGems" or tag.socketColor == "B" and "intelligenceGems"
+					local count = cfg[targetAtrColor] or 0
+					if target == 0 or count < tag.sockets then
```

Comment:
Target is not a local variable here this is probably ment to be something else?

Reply:
Leftover after renaming. Fixed.